acra/acra-in-depth/cryptography-and-key-management/_index.md

```diff
@@ -98,6 +98,10 @@ Additionally, Acra's services accept only TLS 1.2+ connections and cipher suites
 
 ### Password hashing
 
+{{< hint warning >}}
+Deprecated and will not in use after 0.90.0.
```

Comment:
```suggestion
AcraWebConfig tool is deprecated and removed since 0.91.0. After 0.91.0, Acra components don't store any accounts, thus, don't use password hashing.
```